skyportal/tests/frontend/test_data_sharing.py

```diff
@@ -9,6 +12,8 @@ def test_share_data(
     public_group,
     public_group2,
 ):
+    if IS_CI_BUILD:
```

Comment:
Can we reimplement this (and subsequent usages) as a decorator?

Reply:
Rescinding this comment as @acrellin's implementation follows `pytest` best practices: 

https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail